lib/Backend/IR.cpp

```diff
@@ -2859,7 +2860,7 @@ Instr::FindRegUse(StackSym *sym)
         else if (src1->IsIndirOpnd())
         {
             IR::IndirOpnd *indirOpnd = src1->AsIndirOpnd();
-            if (indirOpnd->GetBaseOpnd()->m_sym == sym)
+            if (indirOpnd->GetBaseOpnd() && indirOpnd->GetBaseOpnd()->m_sym == sym)
```

Comment:
Is GetBaseOpnd() const? If not, could you save the result to a variable to avoid a potential deoptimization from the compiler?